src/lib/containers/widgets/query-filter/FacetFilterHeader.tsx

```diff
@@ -1,5 +1,8 @@
 import * as React from 'react'
 import { unCamelCase } from '../../../utils/functions/unCamelCase'
+import iconPlus from '../../../assets/icons/icon_plus.svg'
+import iconMinus from '../../../assets/icons/icon_minus.svg'
+import '../../../style/components/query_filter/_facet-filter-header.scss'
```

Comment:
Instead of importing the scss file here, if you could import it [here](https://github.com/Sage-Bionetworks/Synapse-React-Client/blob/develop/src/lib/style/components/_all.scss)

Reply:
If the stylesheet is shared with other components, I agree it should go to a higher level css file like _all.scss. In this case, these styles are not being shared and I think we can import the scss file here. The reason is that the styles are only imported when the component is rendered and I believe this is preferable. However, if the facet filter component is pretty much on every page, I will import the file in _all.scss instead. Please let me know.